### DISCUSSION TITLE
Learning Js Through Projects

### DISCUSSION THREAD

**Discussion by verma-kunal** [EddieHubCommunity/support#1767] [Show and tell]
Hey everyone!
I'll be starting to learn the concepts in Javascript through building projects & sharing my learnings on socials.
I have created an initial checklist of projects taking reference from FreeCodeCamp article ([Link](https://www.freecodecamp.org/news/javascript-projects-for-beginners/)) & would pave my way through that!
Checklist [here](https://hackmd.io/@verma-kunal/r1EVLYz9_)

Would really love more suggestions & resources to make my learnings fruitful & effective during the period 😃

**Comment by Shanmuganathanb** [comment]
Everytime you are working on a project try to refer to https://developer.mozilla.org/ , Like if you come across a built-in function or method , try to look up its core functionalities. That would better help you to understand them better. And I will also try to follow up on this article and I hope it would be a great value to our progress.
Also if you feel little comfortable with Javascript already have a look at this article https://madasamy.medium.com/javascript-brief-history-and-ecmascript-es6-es7-es8-features-673973394df4 .

Sorry that I have not learned Markup yet.

**Reply by verma-kunal** [reply]
Thank you so much for the resources & suggestions!
Will surely follow them 👍
